src/Network/Receive/ServerType0.pm

```diff
@@ -7386,9 +7386,13 @@ sub msg_string {
 
 sub skill_msg {
 	my ($self, $args) = @_;
-	message TF("id: %s msgid: %s\n", $args->{id}, $args->{msgid}), "info";
 
-	#	'07E6' => ['skill_msg', 'v V', [qw(id msgid)]], #TODO: PACKET_ZC_MSG_SKILL     **msgtable
+	if ($msgTable[++$args->{msgid}]) { # show message from msgstringtable.txt -> [<Skill_Name>] <Message>
```

Comment:
Yes. Simple test, if u ever try my Change Material changes, on success it will shows `id: 2494 msgid: 1574`, in msgstringtable.txt means Successful